### PR TITLE
Russian lang

### PR DESCRIPTION
**Pull request by FKrauss**
Added the two missing strings on the ` bisq-website/_data/languages.yml ` folder

Should be good to go now

### CODE REVIEW

File: ru/images/DAO/dao_benefits.svg [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        <tspan
+           sodipodi:role="line"
+           id="tspan4661"
+           x="472.00201"
+           y="2554.0569">Taxa de</tspan>
+        <tspan
+           sodipodi:role="line"
+           x="472.00201"
+           y="2572.8069"
+           id="tspan4665">negociação</tspan>
```
Comment (RiccardoMasutti) [inline ru/images/DAO/dao_benefits.svg]: This is portuguese, not russian

**Review by RiccardoMasutti — CHANGES_REQUESTED** [review]

**Comment by RiccardoMasutti** [comment]
There are many strings translated incorrectly.

File: ru/images/DAO/dao_benefits.svg (hunk 1 above)
Comment (FKrauss) [reply ru/images/DAO/dao_benefits.svg]: true. nice catch. let me replace with the eng one then. because our Ru translators stopped replying

**Comment by FKrauss** [comment]
Updated the images to the English version as pointed out by @RiccardoMasutti 

we don't have documented exactly where to fetch these strings from Transifex. They probably are translated there but without a way to verify it I rather keep in Eng.

We can plan on translating this at a later PR but I believe being in English is enough to publish

**Review by RiccardoMasutti — APPROVED** [review]
ACK

**Pull request merged**